### ISSUE TITLE
Removal Cutoffs

### ISSUE THREAD

**Issue by sean8300** [sean8300/listerassister_issues#493]
Removals after Friday cutoff are going to Tuesdays, should be Wednesday after 2pm on Fridays 
 Submitted By:  Sean O'Donnell

**Comment by sean8300** [comment]
Slacked management channel:
Hey all, sorry to derail but need clarity on a ticket:
For the removal cutoffs after 2pm Friday it should be going to Wednesday instead of Tuesday, correct?
Is this just for when a date isnt selected and the system decides the next date or is this for the dropdown menu options (or both)?
This is only in Phoenix?

**Comment by sean8300** [comment]
Was told by Nicole in Slack to hold off because things are going to change.  This is in holding pattern, not going anything with it until told otherwise.  Keeping in on deck and in this weeks project, but marking it on hold

**Comment by sean8300** [comment]
This came up in meeting today when they asked for a status.  I didn't have GitHub in front of me so I didnt know the status, didn't realize then that this is in a holding pattern.  After the meeting slacked Chad and Nicole and let them know that this isn't moving because I was told to put it on hold until I'm given more information by Nicole, and I have not yet been given any info.  This remains on hold until I'm told otherwise

**Comment by sean8300** [comment]
We started talking about this again in todays meeting, was told Nicole would update the ticket detailing how the removals work for each market.  So Thursday before 2pm goes to X, Friday after 2pm go to Y, etc

This will go live the week after the 4th (this week).  Waiting on Nicole.  Explained there is already a ticket open for this so they just have to add to this tix, dont have to create a new one, but at this point I'd welcome any information in any form.  Also adding Chad to tix for visibility.

**Comment by sean8300** [comment]
Slack to Greg/Nicole:
Good morning all.  Just a quick reminder that I am waiting for information detailing how removals are to work for each day (Thursday before 2pm goes to X, after 2pm goes to Y, Friday before 2 etc etc).  Please provide this information when available with enough time for me to actually code and test.  Thanks!

**Comment by sean8300** [comment]
Slacked Greg/Nicole again today:
Good morning all. Just a quick reminder that I am waiting for information detailing how removals are to work for each day (Thursday before 2pm goes to X, after 2pm goes to Y, Friday before 2 etc etc).

**Comment by sean8300** [comment]
Have been clear that I need this information with time to be able to react and create, its looking like this is going to turn into one of those "give the information that I've been asking for for weeks 5 minutes before I want it done and then get upset when its not done 'on time'" things.  Not getting it today, tomorrow is the last day of this week because of the holiday....soooo....hopefully get this set up NEXT week?  Its been 29 days since I was told I would be given this info.

**Comment by sean8300** [comment]
Slacked Greg/Nicole again today:
Good morning all. Just a quick reminder that I am waiting for information detailing how removals are to work for each day (Thursday before 2pm goes to X, after 2pm goes to Y, Friday before 2 etc etc).

**Comment by sean8300** [comment]
We are closed tomorrow, which means even if they ended up getting me this information later today it won't be addressed until Monday, four days before this is supposed to be entered and done.  30+ days asking for this info and only give the required info within the last 4 days before its supposed to be done (assuming I actually get the stuff TODAY)

**Comment by sean8300** [comment]
Spoke with Nicole when she came in, she asked "didnt you get the email from Tuesday?".....I was not sent an email on Tuesday.  She said she would send it to me now.

**Comment by sean8300** [comment]
Email from Nicole:
Hi Sean, 

TYPE ONE : 

Market: Phoenix
Any removals that come in Monday before 2 pm ---> Scheduled for Tuesday
Tuesday before 2 pm ---> Wednesday
Wednesday before 2 pm ----> Thursday
Thursday before 2 pm ----> Friday
Friday before 2 pm ----> Saturday 
Saturday after 2 pm ----> Monday

TYPE TWO : 

Market: Tucson, CA, Vegas 
Any removals that come in Monday before 2 pm -----> Scheduled for Tuesday
Tuesday before 2 pm ---> Wednesday
Wednesday before 2 pm ----> Thursday
Thursday before 2 pm ----> Friday
Friday before 2 pm ----> Monday

Please let me know if you have any questions. Thank you!

**Comment by sean8300** [comment]
Finally have info I can work with.  Looking through the info provided and it doesn't look accurate, but I will build out what I've been given.  

Email to Nicole:
Thank you, just double checking that in Phoenix if someone submits a removal after 2pm Saturday it goes to Monday.  For example, if I submit a removal at 9pm Sunday night it will be removed the next morning?

**Comment by sean8300** [comment]
You're right, please see the revision below.

Market: Phoenix
Any removals that come in Monday before 2 pm ---> Scheduled for Tuesday
Tuesday before 2 pm ---> Wednesday
Wednesday before 2 pm ----> Thursday
Thursday before 2 pm ----> Friday
Friday before 2 pm ----> Saturday 
Friday after 2 pm ----> Tuesday

**Comment by sean8300** [comment]
/*Phoenix
                            Monday before 2 = Tuesday
                            Monday after 2 = Wednesday
                            
                            Tuesday before 2 = Wednesday
                            Tuesday after 2 = Thursday
                            
                            Wednesday before 2 = Thursday
                            Wednesday after 2 = Friday
                            
                            Thursday before 2 = Friday
                            Thursday after 2 = Saturday
                            
                            Friday before 2 = Saturday
                            Friday after 2 = Tuesday
                            
                            Saturday = Wednesday
                            Sunday = Wednesday
                             */
                            
                            /* Everyone else
                            Monday before 2 = Tuesday
                            Monday after 2 = Wednesday
                            
                            Tuesday before 2 = Wednesday
                            Tuesday after 2 = Thursday
                            
                            Wednesday before 2 = Thursday
                            Wednesday after 2 = Friday
                            
                            Thursday before 2 = Friday
                            Thursday after 2 = Monday
                            
                            Friday before 2 = Monday
                            Friday after 2 = Tuesday
                            
                            Saturday = Tuesday
                            Sunday = Tuesday
                             */

**Comment by sean8300** [comment]
Nicole slacked asking that Phoenix Saturdays go to Tuesday

**Comment by sean8300** [comment]
/*Phoenix
                            Monday before 2 = Tuesday
                            Monday after 2 = Wednesday

                            Tuesday before 2 = Wednesday
                            Tuesday after 2 = Thursday

                            Wednesday before 2 = Thursday
                            Wednesday after 2 = Friday

                            Thursday before 2 = Friday
                            Thursday after 2 = Saturday

                            Friday before 2 = Saturday
                            Friday after 2 = Tuesday

                            Saturday = Tuesday
                            Sunday = Wednesday
                             */

                            /* Everyone else
                            Monday before 2 = Tuesday
                            Monday after 2 = Wednesday

                            Tuesday before 2 = Wednesday
                            Tuesday after 2 = Thursday

                            Wednesday before 2 = Thursday
                            Wednesday after 2 = Friday

                            Thursday before 2 = Friday
                            Thursday after 2 = Monday

                            Friday before 2 = Monday
                            Friday after 2 = Tuesday

                            Saturday = Tuesday
                            Sunday = Tuesday
                             */

**Comment by sean8300** [comment]
Adding to this work load, have fixed the issue where you couldn't shut down a specific date, that now works.  Have also built it out so there is a list of company wide holidays and now market specific holidays, so if we wanted to shut down just Phoenix but leave everyone else open we can now do that
